src/main/java/com/laserfiche/repository/api/clients/impl/TasksApiEx.java

```diff
@@ -0,0 +1,20 @@
+package com.laserfiche.repository.api.clients.impl;
+
+import retrofit2.Response;
+import retrofit2.http.DELETE;
+
+import java.util.concurrent.CompletableFuture;
+
+public interface TasksApiEx {
+    /**
+     *
+     * - Cancels an operation. - Provide an operationToken to cancel the operation, if possible. Should be used if an operation was created in error, or is no longer necessary. - Rollbacks must be done manually. For example, if a copy operation is started and is halfway complete when canceled, the client application is responsible for cleaning up the files that were successfully copied before the operation was canceled.
+     * @param repoId The requested repository ID (required)
+     * @param operationToken The operation token (required)
+     * @return CompletableFuture&lt;ResponseBody&gt;
```

Comment:
Should the type be exactly `CompletableFuture<Response<Void>>` or is `CompletableFuture<ResponseBody>` the same thing?

Reply:
No they are not. I will update.

Reply:
Fixed. Thanks.